tap_github.py

```diff
@@ -9,15 +9,16 @@
 logger = singer.get_logger()
 
 
-def authed_get(source, url):
+def authed_get(source, url, headers={}):
     with singer.stats.Timer(source=source) as stats:
+        session.headers.update(headers)
```

Comment:
Oh wait, I see you're resetting them every time - so that isn't as much of a concern